From c60e548d413fe20495a53d6e5059d3805cb8f00b Mon Sep 17 00:00:00 2001
From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 08:35:05 -0700
Subject: Use a local copy of object.inv from doc.python.org and
 cairographics.org, instead of downloading them each time from the internet

Last-Update: 2014-09-22

Patch-Name: use_local_objects.inv.patch
---
 docs/conf.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 345dcfb..2248a2d 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -18,5 +18,5 @@ exclude_patterns = ['_build']
 autodoc_member_order = 'bysource'
 autodoc_default_flags = ['members']
 intersphinx_mapping = {
-    'http://docs.python.org/': None,
-    'http://cairographics.org/documentation/pycairo/2/': None}
+    'http://docs.python.org/': '../debian/python.org_objects.inv',
+    'http://cairographics.org/documentation/pycairo/2/': '../debian/cairographics.org_pycairo2_objects.inv'}
